CHANGELOG.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+1.1.0 (2022-03-23)
```

Comment:
We can merged `1.1.0a2` section completely now with only 1.1.0

Reply:
Should we also include `BigQueryTableExistenceSensorAsync` in the list of new operators that was merged today - https://github.com/astronomer/astronomer-providers/pull/135

Reply:
The example DAG for this sensor is currently failing integration test (it ran fine in local though), hence didn't put it yet.

Reply:
Included BigQueryTableExistenceSensorAsync and fix to malformed example DAG URLs in the latest commit